.projen/tasks.json

```diff
@@ -121,9 +118,22 @@
     },
     "install": {
       "name": "install",
+      "description": "Install project dependencies and update lockfile (non-frozen)",
+      "steps": [
+        {
+          "exec": "pnpm i --no-frozen-lockfile"
+        },
+        {
+          "exec": "pnpx nx run-many --target install-py --projects pipeline-sample-py --parallel=1"
+        }
+      ]
+    },
+    "install:ci": {
+      "name": "install:ci",
+      "description": "Install project dependencies using frozen lockfile",
       "steps": [
         {
-          "exec": "npx nx run-many --target install-py --projects pipeline-sample-py --parallel=1"
+          "exec": "pnpm i --frozen-lockfile"
         }
       ]
     },
```

Comment:
So in `install:ci` now we do not need to `install-py` within projects? How will py deps get installed during CI?